bochs/gui/textconfig.cc

```diff
@@ -202,12 +202,12 @@ int ask_int(const char *prompt, const char *help, Bit32s min, Bit32s max, Bit32s
     }
     if ((clean[0] == '?') && (strlen(help) > 0)) {
       bx_printf("\n%s\n", help);
-      bx_printf("Your choice must be an integer between %u and %u.\n\n", min, max);
+      bx_printf("Your choice must be an integer between %ld and %ld.\n\n", min, max);
```

Comment:
``long`` is a 32-bit type on some platforms, ``%l_`` is definitely not enough for 64-bit values.

Reply:
Using the portable symbol FMT_LL (from osdep.h) should fix this issue.